test/helpers/view_components/view_component_test_case.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class ViewComponentTestCase < ActionView::TestCase
```

Comment:
I've created this helper class for the ActionView tests to allow signing in the current user, which I needed for the mentor header test

Reply:
:+1:

Reply:
I'd consider adding this to `ActionView::TestCase` itself I think. (See how we do the same for integration tests in `test_helper.rb`). The its available everywhere rather than having to change the base class.